packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```
@@ -41,11 +54,19 @@ BuildRequires: %{_cross_os}kernel-6.1-archive
 %description
 %{summary}.
 
+%package -n fabricmanager
+Summary: NVIDIA fabricmanager config and service files
+Version: 1.0.0
+License: %{spdx_id}
```

You don't need to repeat license or version if they should be inherited from the top-level package.

Also, it would be good to make it so this package depends on the right things - it doesn't work without some source of binaries, which then the tesla subpackage can provide:
```suggestion
Requires: %{name}-tesla(fabric-manager)
```